### PR TITLE
[LOOM-1284][BpkFlare] Fix class 2 CSS issues for BpkFlare

### PR DESCRIPTION
**Pull request by metalix2**
Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [x] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: packages/bpk-component-flare/src/bpk-flare-bar.module.scss [hunk 1]
```diff
@@ -42,7 +41,8 @@
     // stylelint-disable-next-line unit-disallowed-list
     bottom: -1px;
     width: 700rem; // required for correct behaviour in IE
-    height: tokens.$bpk-flare-height-desktop;
+    // stylelint-disable-next-line unit-disallowed-list
+    height: calc(tokens.$bpk-flare-height-desktop - 1px);
```
Comment (metalix2) [inline packages/bpk-component-flare/src/bpk-flare-bar.module.scss:45]: Fixes bleeding from the bottom -1px

File: packages/bpk-component-flare/src/bpk-flare-bar.module.scss [hunk 2]
```diff
@@ -56,20 +56,4 @@
       height: tokens.$bpk-flare-height-mobile;
     }
   }
-
-  &__rounded-corner {
```
Comment (metalix2) [inline packages/bpk-component-flare/src/bpk-flare-bar.module.scss:60]: Moved to the bpk-content-bubble scss file

File: packages/bpk-component-flare/src/bpk-content-bubble.module.scss [hunk 3]
```diff
@@ -44,6 +46,7 @@
   }
 
   &__container {
+    position: relative;
```
Comment (metalix2) [inline packages/bpk-component-flare/src/bpk-content-bubble.module.scss:49]: Allows us to use absolute positioning for the rounded-corners.

File: packages/bpk-component-flare/src/BpkContentBubble.js [hunk 4]
```diff
@@ -61,6 +62,18 @@ const BpkContentBubble = (props) => {
...
   }
+  const leftCornerRadiusClassNames = [
+    getClassName('bpk-content-bubble__rounded-corner'),
+  ];
+  const rightCornerRadiusClassNames = [
+    getClassName('bpk-content-bubble__rounded-corner'),
+    getClassName('bpk-content-bubble__rounded-corner--trailing'),
+  ];
+
+  if (flareProps?.svgClassName) {
```
Comment (metalix2) [inline packages/bpk-component-flare/src/BpkContentBubble.js:73]: Maintain existing props for the corner svgs

File: packages/bpk-component-flare/src/BpkContentBubble.js [hunk 5]
```diff
@@ -69,15 +82,23 @@ const BpkContentBubble = (props) => {
   return (
     <div className={wrapperClassNames.join(' ')} {...rest}>
       <div className={getClassName('bpk-content-bubble__container')}>
-        <div className={contentClassNames.join(' ')}>{content && content}</div>
+        <div className={contentClassNames.join(' ')}>
+          {content && content}
```
Comment (mungodewar) [inline packages/bpk-component-flare/src/BpkContentBubble.js:86]:
```suggestion
          {content}
```

File: packages/bpk-component-flare/src/BpkContentBubble.js [hunk 6]
```diff
@@ -69,15 +82,23 @@ const BpkContentBubble = (props) => {
...
+          {/* These rounded svgs are required as the container background extends into the flare meaning that when we "round" the container the rounded bottom corners are hidden by the flare. */}
+          {rounded && showPointer && (
+          <CornerRadius className={leftCornerRadiusClassNames.join(' ')} />
+          )}
+          {rounded && showPointer && (
+            <CornerRadius className={rightCornerRadiusClassNames.join(' ')} />
+          )}
```
Comment (mungodewar) [inline packages/bpk-component-flare/src/BpkContentBubble.js:93]:
```suggestion
          {rounded && showPointer && (
          <CornerRadius className={leftCornerRadiusClassNames.join(' ')} />
          <CornerRadius className={rightCornerRadiusClassNames.join(' ')} />
          )}
```

**Review by mungodewar — APPROVED** [review]
LGTM

**Pull request merged**